zulia-data/src/main/java/io/zulia/data/input/FileDataInputStream.java

```diff
@@ -1,13 +1,8 @@
 package io.zulia.data.input;
 
-import io.zulia.data.DataStreamMeta;
-
-import java.io.BufferedInputStream;
-import java.io.File;
-import java.io.FileInputStream;
-import java.io.FileNotFoundException;
-import java.io.IOException;
-import java.io.InputStream;
+import io.zulia.data.common.DataStreamMeta;
+
+import java.io.*;
```

Comment:
Was this intentional or a bad IntelliJ setting caused it to over-import?

Reply:
Bad intellij